src/app/order/order-detail/order-detail-card/order-detail-card.component.ts

```diff
@@ -9,7 +12,42 @@ import { Order } from "@boklisten/bl-model";
 export class OrderDetailCardComponent implements OnInit {
 	@Input() order: Order;
 
-	constructor() {}
+	public selectedBranch: string;
+	public branches: Branch[];
+	public isAdmin: boolean;
 
-	ngOnInit() {}
+	constructor(
+		private _branchService: BranchService,
+		private _orderService: OrderService,
+		private _toasterService: ToasterService,
+		private _authService: AuthService
+	) {}
+
+	ngOnInit() {
+		this.isAdmin = this._authService.isAdmin();
+		this.getAllBranches();
+		this.selectedBranch = String(this.order.branch);
+		console.log(this.selectedBranch);
+	}
+
+	private async getAllBranches() {
+		this._branchService.get().then((branches: Branch[]) => {
+			this.branches = branches;
+		});
+	}
+
+	public async updateOrderBranch(newBranchId: string) {
```

Comment:
I understand it's only necessary to update the order branch for P2P-matching, but would that not have any unintended consequences when the order disagrees with the customeritem? I trust your answer so you can merge if you want to, I just want to know you've thought about it

Reply:
That is a very good point! I updated it so it updates the customerItems as well. (that handoutById (branch) on the customerItem is used for reminders)